spec.html

```diff
@@ -107,6 +107,30 @@ <h1>Well-Known Intrinsic Objects</h1>
 <emu-clause id="sec-abstract-operations">
   <h1>Abstract Operations</h1>
 
+  <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+    <h1>
+      SetterThatIgnoresPrototypeProperties (
+        _home_: an Object,
```

Comment:
Is it intentional for this parameter name to evoke [[HomeObject]]? The phrasing “On the home_object ” in particular strongly suggests that to me.

It does kind of of align with an intuitive notion of what the hypothetical [[HomeObject]] values of these setters could have been set to (i.e., if built-in functions had that slot). The connection is pretty tenuous though — ES code polyfills for these functions, for example, would be very unlikely to end up with %Iterator.prototype% as their “real” [[HomeObject]] values. Being a merely near conceptual match actually seems to me like a good reason not to use the name. (Maybe something like “intrinsicOwner” or even just “proto”?)

The naming of an AO variable isn’t very important, of course — just a thought I had when reading the updated text (which seems much clearer now btw).

Reply:
I was aware of the non-uniqueness of the name, but when I asked myself "what is the name of the place where this property lives?", the obvious answer was "home". That is probably also how `[[HomeObject]]` got its name. But I don't think the alias name alone will imply a strong relationship to `[[HomeObject]]`. I'm open to alternative suggestions if you've got them.